pynamodb/models.py

```diff
@@ -193,8 +192,6 @@ class MetaModel(AttributeContainerMeta):
     billing_mode: Optional[str]
     stream_view_type: Optional[str]
 
-    _version_attribute_name: str
```

Comment:
This was plain wrong. It's a property of the Model type (and it's indeed defined there too), not the MetaModel type.